staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -35,6 +35,10 @@ const (
 	// apiserver and job controller will ignore this annotation and they will
 	// always track jobs using finalizers.
 	JobTrackingFinalizer = "batch.kubernetes.io/job-tracking"
+	// The Job labels will use batch.kubernetes.io as a prefix for all labels
+	// Historically the job controller uses unprefixed labels for job-name and controller-uid and
+	// Kubernetes continutes to recognize those unprefixed labels for consistency.
+	JobLabelPrefix = "batch.kubernetes.io/"
```

Comment:
Shouldn't we define these sorts of cross-version constants in `staging/src/k8s.io/api/batch` rather than `.../v1` ?  I know it's not exactly common, but having the same const defined twice feels like a smell to me.

Reply:
this is a recurring topic of debates in API reviews :)

Someone writing a controller should use the versioned packages. So we actually should have them here.

Reply:
My point was that it seems unlikely that such a key woyuld change across API versions.

I'll resolve this and discuss with other API reviewers async.